File: market/js/AllTokensVolumeRankingTimePeriod.js
var request = require("request");
// set parameters
const startDate = "startDate=1577836800000";
const endDate = "endDate=1577923200000";
// set url
const url = "https://api.deversifi.com/v1/pub/USDRanking?" + startDate + "&" + endDate;
// execute request and print result
request(url, function(error, response, body) {
    console.log(JSON.parse(body));
});

File: market/python/AllTokensVolumeRankingTimePeriod.py
import requests
import json

# set the url
url = "https://api.deversifi.com/v1/pub/USDRanking"

# give start- and enddates as params
params = {
    "startDate": "1577836800000",
    "endDate": "1577923200000"
}

# standard headers
headers = {
    "Accept": "application/json",
    "Content-Type": "application/json"
}

# execute the request and load into dictionary
response = json.loads(requests.request(
    "GET", url, headers=headers, params=params).text)

# print the response from dictionary
print(json.dumps(response, indent=2))
